api/apiCalls.js

```diff
@@ -0,0 +1,37 @@
+export function deleteData(path, id) {
+	return fetch(`${path}/${id}`, {
+		method: 'DELETE',
+	})
+}
+
+export function fetchData(path) {
+	return fetch(`/${path}`)
```

Comment:
This function has little meaning as it's not doing anything extra other than native `fetch`.
Also, could you look into maintaining consistency and either using the preceding `/` for urls or not (only the delete one doesn't start with it).